src/components/reception/MultiplexedLibraryBarcode.vue

```diff
@@ -135,6 +139,18 @@ const props = defineProps({
     type: Object,
     default: () => ({}),
   },
+  additionalDetails: {
```

Comment:
Looks like this is duplicated in several places? Not sure if it is worth abstracting out but worth considering.

Reply:
MultiplexedLibraryBarcode and The MultiBarcode component have the same defined properties as they are injected into the main component dynamically